LayoutTests/fast/webgpu/repro_294602-expected.txt

```diff
@@ -0,0 +1,11 @@
+CONSOLE MESSAGE: Blocked access to external URL https://webgpufundamentals.org/webgpu/resources/webgpu-lesson.css
```

Comment:
test case is currently only runnable locally, not sure how to otherwise measure the performance.

Existing CTS tests ensure correctness